apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/GrpcRdaSource.java

```diff
@@ -236,17 +236,19 @@ private void logOrRethrowDroppedConnectionException(
   /**
    * Uses the hard coded starting sequence number if one has been configured. Otherwise asks the
    * sink to provide its maximum known sequence number as our starting point. When all else fails we
-   * start at the beginning.
+   * start at the beginning. The RDA API assumes we are sending the highest sequence number that we
+   * already have so they will start at that plus 1. For a configured starting sequence number we
+   * subtract one but for a value from the database we can just pass it through unchanged.
    *
    * @param sink used to obtain the maximum known sequence number
    * @return a valid RDA change sequence number
    */
   private long getStartingSequenceNumber(RdaSink<TMessage, TClaim> sink)
       throws ProcessingException {
     if (startingSequenceNumber.isPresent()) {
-      return startingSequenceNumber.get();
+      return startingSequenceNumber.map(x -> Math.max(MIN_SEQUENCE_NUM, x - 1)).get();
```

Comment:
I'm wondering if it makes more sense to leave this as it was and make our variable align directly with how the RDA API is interpreting it.

Reply:
No the `Math.max()` call ensures the value can't be lower than zero.